### PR TITLE
Updated MacOS usb-serial info for the T-Echo

### PR DESCRIPTION
**Pull request by ventz**
* Specifically around the usb serial/TTY information.
* A note about not needing (and actually having to remove the drivers/kernel extension)

### CODE REVIEW

File: website/docs/getting-started/flashing-nrf52.md [hunk 1]
```diff
@@ -8,12 +8,44 @@ import TabItem from '@theme/TabItem';
...
+
+Confirm that you have a usb-c cable with "data" lines -- many cables (ie: The one that comes from LilyGo, Anker, etc are "power" only)
+
+#### Windows:
+
 You may need to install the [USB device drivers](http://www.wch-ic.com/search?q=ch340&t=downloads) if your device does not show up when connected.
 
+#### Mac OS
+
+(Verified as of 11/2021 on the latest T-Echo nRF52840 on Mac OS Monterey v12.0.1 - Intel chipset)
```
Comment (sachaw) [inline website/docs/getting-started/flashing-nrf52.md:27]: Don't need a timestamp, as these steps should be as timeless as possible.

File: website/docs/getting-started/flashing-nrf52.md [hunk 2]
```diff
@@ -8,12 +8,44 @@ import TabItem from '@theme/TabItem';
 
 ## Pre-requisits
 
-Please ensure that you use a data USB cable, as some cables provide power only and not the data lines.
+Please ensure that you use a data USB cable, as some cables provide power only and not the data lines. Many popular brands (ie: Anker) are also "power" only.
```
Comment (sachaw) [inline website/docs/getting-started/flashing-nrf52.md:11]: Not really needed, inferred by previous comment.

File: website/docs/getting-started/flashing-nrf52.md [hunk 3]
```diff
@@ -8,12 +8,44 @@ import TabItem from '@theme/TabItem';
...
+
 You may need to install the [USB device drivers](http://www.wch-ic.com/search?q=ch340&t=downloads) if your device does not show up when connected.
 
+#### Mac OS
+
+(Verified as of 11/2021 on the latest T-Echo nRF52840 on Mac OS Monterey v12.0.1 - Intel chipset)
+
+If you have it, you can use the Apple (iPad Pro included) usb-C to usb-C cable, as it has "data" lines. The cable provided from LILYGO will not work, as it's power only.
+
+CRITICAL: *DO NOT* download the USB device drivers - this will actually prevent you from connecting to your T-Echo from your Mac.
```
Comment (sachaw) [inline website/docs/getting-started/flashing-nrf52.md:31]: This is a good place for an admonition:
https://docusaurus.io/docs/markdown-features/admonitions

File: website/docs/getting-started/flashing-nrf52.md [hunk 4]
```diff
@@ -8,12 +8,44 @@ import TabItem from '@theme/TabItem';
...
+
+If you are starting clean (or have removed the WCH USB driver), verify successful connections with 
+
+* Plug in your T-Echo
+* Open the Terminal
+* `ls -l /dev/tty.usbmodem*`
+
+If the device file exists, you will also notice a "TECHOBOOT" volume in the Finder, or in the Terminal after double-clicking the Reset Button (see below) 
+
+* `ls /Volumes/TECHOBOOT`
```
Comment (sachaw) [inline website/docs/getting-started/flashing-nrf52.md:47]: Might be a good case to use `details`: https://docusaurus.io/docs/markdown-features#details
for edge cases as users are most likely to want to read them if they are looking for it in the first place.

File: website/docs/getting-started/flashing-nrf52.md [hunk 5]
```diff
@@ -33,6 +65,9 @@ Be careful to install the correct load for your board. While it is unlikely that
 
 * Connect your device to your computer with a USB cable. If you computer complains about needing to format a new drive, cancel the format command.
 * Double click the reset button on your device, this will put it into boot loader mode.
+
+[<img alt="LILYGO T-Echo" src="/img/hardware/t-echo-lilygo.jpg" style={{zoom:'25%'}}  />](/img/hardware/t-echo-lilygo.jpg)
+
 * A new drive will then be mounted on your computer. Open this drive and you should see three files: `CURRENT.UF2`, `INDEX.HTM`, and `INFO_UF2.TXT`
-* Copy the appropriate `firmware-xxxxx-1.2.x.uf2` file from the firmware zip file onto the new drive.
+* Copy the appropriate `firmware-xxxxx-1.2.x.uf2` file from the firmware zip file onto the new drive. NOTE: You are going to copy/drop "as is" 'firmware-xxxxx-1.2.x.uf2' (NOT over "CURRENT.UF2") in the volume, and the device reboot will copy it/load it correctly.
```
Comment (sachaw) [inline website/docs/getting-started/flashing-nrf52.md:72]: Another place for an admonition.

**Review by sachaw — CHANGES_REQUESTED** [review]
Looks good, just a few pointers to fix up, then I can merge.

File: website/docs/getting-started/flashing-nrf52.md (hunk 1 above)
Comment (ventz) [reply website/docs/getting-started/flashing-nrf52.md]: Done - cleaned up (overwrote latest push - see new file)

File: website/docs/getting-started/flashing-nrf52.md (hunk 2 above)
Comment (ventz) [reply website/docs/getting-started/flashing-nrf52.md]: Done - cleaned up (overwrote latest push - see new file)

File: website/docs/getting-started/flashing-nrf52.md (hunk 3 above)
Comment (ventz) [reply website/docs/getting-started/flashing-nrf52.md]: Done - added + cleaned up (overwrote latest push - see new file)

File: website/docs/getting-started/flashing-nrf52.md (hunk 4 above)
Comment (ventz) [reply website/docs/getting-started/flashing-nrf52.md]: Done - added detail block (overwrote latest push - see new file)

File: website/docs/getting-started/flashing-nrf52.md (hunk 5 above)
Comment (ventz) [reply website/docs/getting-started/flashing-nrf52.md]: Done - added (overwrote latest push - see new file)

**Comment by ventz** [comment]
@sachaw Thanks for the detailed feedback. Cleaned up, updated (with admonitions and details), and re-pushed (force updated since my push is not used upstream anywhere yet)

**Pull request merged**